src/System.Runtime/tests/System/Int32Tests.cs

```diff
@@ -151,130 +151,238 @@ public static void ToString_InvalidFormat_ThrowsFormatException()
         public static IEnumerable<object[]> Parse_Valid_TestData()
         {
             NumberStyles defaultStyle = NumberStyles.Integer;
-            NumberFormatInfo emptyFormat = new NumberFormatInfo();
-
-            NumberFormatInfo customFormat = new NumberFormatInfo();
-            customFormat.CurrencySymbol = "$";
-
-            yield return new object[] { "-2147483648", defaultStyle, null, -2147483648 };
```

Comment:
In fact, several of these tests that were using NumberStyles.Integer are no longer testing that style.